instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/UpcomingRouteObjectsTest.kt

```diff
@@ -537,6 +537,88 @@ class UpcomingRouteObjectsTest : BaseCoreNoCleanUpTest() {
         checkRoadObjects(expectedObjectsAfterFirstRefresh, updateAfterRefresh.upcomingRoadObjects)
     }
 
+    @Test
+    fun duplicateIncidentIdsTest() = sdkTest {
```

Comment:
Is it the same as from the other PR: https://github.com/mapbox/mapbox-navigation-android/pull/7424/files#diff-96d2751df96c03f92acb2f4490ff03a01b092596728b1e474ebe21e5105599fcR542 ?

Reply:
Yes, it shouldn't be here.